specs/mechanics/training.md

```
@@ -0,0 +1,93 @@
+
+Move learned is selected from eligible moves based on:
+- Facility type
+- Pet's current stats
+- Species
+
+See [Battle](./battle.md) for move details.
+
+## Stat Caps
+
+Maximum trainable stat values increase with growth stages. Caps are defined as:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)